src/chartLibraries/dygraph/index.js

```diff
@@ -48,6 +48,7 @@ export default (sdk, chart) => {
 
     executeLatest = makeExecuteLatest()
     const isEmpty = attributes.outOfLimits || result.data.length === 0
+    const { disabledYAxis } = chart.getAttributes()
```

Comment:
Do we need to add this into initialAttributes?

Reply:
@jjtsou Yes! I didn't realise we have this file, maybe now i'll be able to change it to `enabledYAxis` with default `true`